qa/L0_batcher/batcher_test.py

```diff
@@ -151,6 +152,26 @@ def test_static_batch_preferred(self):
             except InferenceServerException as ex:
                 self.assertTrue(False, "unexpected error {}".format(ex))
 
+    def test_static_batch_preferred_shm(self):
```

Comment:
Only this case is tested with shared memory?

Reply:
Modified for greater test coverage.